plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogConfig.java

```diff
@@ -68,4 +74,69 @@ public IcebergNessieCatalogConfig setDefaultWarehouseDir(String defaultWarehouse
         this.defaultWarehouseDir = defaultWarehouseDir;
         return this;
     }
+
+    @Config("iceberg.nessie-catalog.auth.type")
+    @ConfigDescription("The authentication type to use. Available values are NONE | BEARER. Default: NONE")
+    public IcebergNessieCatalogConfig setAuthenticationType(AuthenticationType authenticationType)
+    {
+        this.authenticationType = Optional.ofNullable(authenticationType);
+        return this;
+    }
+
+    public Optional<AuthenticationType> getAuthenticationType()
+    {
+        return authenticationType;
+    }
+
+    @Config("iceberg.nessie-catalog.auth.bearer.token")
```

Comment:
WDYT about using config names similar to the Nessie java client's?.. e.g.: `iceberg.nessie-catalog.authentication.token` to match https://github.com/projectnessie/nessie/blob/c286584ff149b38ffd7815a207a7529d27020a6c/api/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java#L49